keyvault/client/secrets/src/samples/java/secrets/ListOperations.java

```diff
@@ -28,16 +30,16 @@ public static void main(String[] args) throws IllegalArgumentException {
                 .credential(new AzureCredential())
                 .build();
 
-        // Let's create secrets holding storage and bank accounts credentials valid for 1 year. if the secret
+        // Let's create keys holding storage and bank accounts credentials valid for 1 year. if the secret
```

Comment:
Your string replace has done too much, hasn't it? This is still a secrets sample?